auto_dev/services/eject/index.py

```diff
@@ -856,22 +849,6 @@ def _update_and_cleanup(self) -> None:
         self.update_config(agent_config_path, self.config.fork_id.author, self.config.fork_id.name, "agent")
         self.cleanup_directories()
 
-    def cleanup_directories(self) -> None:
```

Comment:
Wut! 
You just clear out everything?!?! I dont really think eject is part of publish ahaha